topics/valkey-bundle.md

```diff
@@ -0,0 +1,173 @@
+2. Valkey Bloom - Provides space-efficient probabilistic data structures, such as Bloom filters, for adding elements and checking whether they exist in a set.
+3. Valkey Search - Enables the creation of indexes and similarity searches through the use of full-text and complex filters. 
+4. Valkey LDAP - Handles user authentication against LDAP based identity providers.  
+
+# Quick Start to Using the Bundle
+
+The fastest way to start using Valkey Bundle is by downloading the official image through [Docker Hub](https://hub.docker.com/r/valkey/valkey-extensions/). The following steps will guide you through launching and interacting with your first instance!
+
+1. **Pull the image to get the latest public release**
+
+    ```bash
+    docker pull valkey/valkey-extensions
+    ```
+
+    This command downloads the most recent stable image of the Valkey Bundle, which includes the Valkey server along with the preloaded modules.
+
+    The Valkey Bundle image also supports multiple tags, allowing you to control the specific version and operating system base. This allows for more control over the environment, whether you’re aiming for a reproducible build (using a version like 8.1-bookworm) or a minimal footprint (alpine variant).
+
+    Check out the [Valkey Bundle Docker Hub Tag](https://hub.docker.com/r/valkey/valkey-extensions/tags) section to view all available tags and example pull commands. 
+
+2. **Start the Valkey server inside a container**
+
+    ```bash
+    docker run --name my-valkey-extensions -d -p 6379:6379 valkey/valkey-extensions
+    ```
+
+    This command starts the Valkey server in a background container named my-valkey-extensions and maps Valkey’s default port 6379 to your local machine for external access. By default, it uses the latest available image. To run a specific version or variant, append the desired tag to the image name. For example:
+
+    ```bash
+    docker run --name my-valkey-extensions -d -p 6379:6379 valkey/valkey-extensions:8.1-bookworm
+    ```
+ 
+3. **Connect to the server using valkey-cli**
+    
+    To interact with the server, use the Valkey command-line interface (valkey-cli). The example below runs the CLI from a separate container on the same Docker network.
+     
+    ```bash 
+    docker run -it --network some-network --rm valkey/valkey-extensions valkey-cli -h my-valkey-extensions
+    ```
+    
+    This approach is ideal when you want to isolate your CLI session or connect from other containers. However, it does require both containers to be connected to the same Docker network.
+
+    Alternatively, you can run the CLI directly inside the server container: 
+
+    ```bash
+    docker exec -it my-valkey-extensions valkey-cli
+    ```
+
+    This runs the valkey-cli directly inside the server container and connects to the server on localhost. This approach requires no extra setup but only works when the server container is running locally and accessible from your host environment.
```

Comment:
```suggestion
    This runs the valkey-cli directly inside the server container and connects to the server on localhost. This approach requires no extra setup but only works when the server container is running locally and accessible from your host environment.
```

As written you are right, but the docker daemon can be configured to be accessible on another host.